wweval/DESCRIPTION

```diff
@@ -89,7 +88,6 @@ Roxygen: list(markdown = TRUE)
 Config/Needs/check: rcmdcheck, testthat
 RoxygenNote: 7.3.2
 Remotes:
-    cfaforecastrenewalww=CDCgov/wastewater-informed-covid-forecasting/cfaforecastrenewalww@less-cfaforecastrenewalww-2,
     stan-dev/cmdstanr,
     wwinference=CDCgov/ww-inference-model
```

Comment:
Should we point to a release version or something? Right now, it will download the dev version. Do we want that?

Reply:
I think we should point to a release! I will add